### PR TITLE
S2U-5 5.1.1.1 Tests & Quizzes: Dynamic Rubrics - MASTER

### PR DESCRIPTION
**Pull request by stetsche**
Conversion scripts: #234

https://sakaiproject.atlassian.net/browse/S2U-5

### CODE REVIEW

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-association.js [hunk 1]
```diff
@@ -281,4 +294,4 @@ class SakaiRubricAssociation extends RubricsElement {
 
 }
 
-customElements.define("sakai-rubric-association", SakaiRubricAssociation);
+customElements.define("sakai-rubric-association", SakaiRubricAssociation);
```
Comment (bgarciaentornos) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-association.js:297]: EOF

**Review by bgarciaentornos — CHANGES_REQUESTED** [review]
Thanks, a few minor things related to the S2U-5 commit.

NOTE: This other fix will also have to be incorporated

File: rubrics/impl/src/main/java/org/sakaiproject/rubrics/impl/repository/AssociationRepositoryImpl.java [hunk 2]
```diff
@@ -59,6 +62,7 @@ public Optional<ToolItemRubricAssociation> findByToolIdAndItemId(String toolId,
     public Optional<ToolItemRubricAssociation> findByItemIdAndRubricId(String itemId, Long rubricId) {
 
         Session session = sessionFactory.getCurrentSession();
+        log.info("itemId {}, rubricId {}", itemId, rubricId);
```
[the remark this answers is not in the document]
Comment (stetsche) [reply rubrics/impl/src/main/java/org/sakaiproject/rubrics/impl/repository/AssociationRepositoryImpl.java:65]: I'd prefer to remove it

File: samigo/samigo-app/src/webapp/jsf/evaluation/gradeStudentResult.jsp [hunk 3]
```diff
@@ -38,6 +38,7 @@ $Id$
 
     <script src="/samigo-app/jsf/widget/hideDivision/hideDivision.js"></script>
     <script src="/library/webjars/jquery/1.12.4/jquery.min.js"></script>
+    <script src="/library/webjars/jquery-ui/1.12.1/jquery-ui.min.js"></script>
```
[the remark this answers is not in the document]
Comment (stetsche) [reply samigo/samigo-app/src/webapp/jsf/evaluation/gradeStudentResult.jsp:41]: Yes, I needed to add it for the modal dialog to work

**Comment by stetsche** [comment]
Thanks @bgarciaentornos! I will make the adjustments and cherry-pick the other fix when I rebase after S2U-32/28 is merged.

**Comment by stetsche** [comment]
I applied the fixes and rebased onto master. When Build is passing, this should be ready to go!

**Review by mpellicer — APPROVED** [review]
This is awesome, terrific job @stetsche and @bgarciaentornos.

Merging this after #11789 as we agreed.

**Pull request merged**